pkg/pods/pod_manifest.go

```diff
@@ -28,7 +28,7 @@ type LaunchableStanza struct {
 type PodManifest struct {
 	Id                string                      `yaml:"id"` // public for yaml marshaling access. Use ID() instead.
 	LaunchableStanzas map[string]LaunchableStanza `yaml:"launchables"`
-	Config            map[string]interface{}      `yaml:"config"`
+	Config            map[interface{}]interface{} `yaml:"config"`
```

Comment:
There's nothing wrong with an application specifying an integer key in their own config, so I'd rather not coerce it to a string.